packages/send-and-read-emails/client/react/src/ReadEmails.js

```diff
@@ -0,0 +1,49 @@
+import { useNylas } from '@nylas/nylas-react';
+import { useState } from 'react';
+import DisplayEmail from './DisplayEmail';
+import { styles } from './styles';
+
+function ReadEmails({ userId }) {
+  const nylas = useNylas();
+  const [threads, setThreads] = useState([]);
+
+  const fetchEmails = async () => {
+    if (!userId) {
+      return;
+    }
+
+    const url = nylas.serverBaseUrl + '/nylas/read-emails';
+
+    const requestHeaders = new Headers();
+    requestHeaders.set('Authorization', userId);
+    requestHeaders.set('Content-Type', 'application/json');
+
+    const res = await fetch(url, {
+      method: 'GET',
+      headers: requestHeaders,
+    });
+
+    const data = await res.json();
+
+    setThreads(data.threads);
```

Comment:
I got this when I ran the app
<img width="794" alt="Screen Shot 2022-09-13 at 1 55 14 PM" src="https://user-images.githubusercontent.com/55773810/189975592-2b06499d-de00-40bd-8da1-a07cfbdd296d.png">.

```suggestion
    setThreads(data);
```

Reply:
That's odd, valid response has `thread` key. 
![image](https://user-images.githubusercontent.com/34139730/189990910-724cb00d-67c6-4954-9d88-3bf5715841a3.png)

Reply:
Is it possible that there's a discrepancy between the response for `node-express` and `node`?

Reply:
@heathergh yep, just updated it! https://github.com/nylas/use-cases/pull/29/files#diff-aa259602b1444137c2cea570347f3aa748c275c564eaa14a078c4d450c45e78fR38

Reply:
Good catch!